### ISSUE TITLE
Feat: Performance ratings and Dojo Analytics

### ISSUE THREAD

**Issue by jalpp** [jackstenglein/chess-dojo-scheduler#587]
Kostya: 
Personally would want to start with performance ratings - with white, with black, vs e4, vs d4, and against opponents of different rating levels (within 100 points, +100, +200, -100, -200, etc.)

Performance rating formula I believe is just (if won, then opp rating +400, loss = opp rating -400, and draw = opp rating as is)

And for multiple games you just take the average of the individual performance rating of each game

**Comment by LucaGiancristofaro** [comment]
@jackstenglein , it seems to me that most of this request is obsolete and already covered in games and repertoire spy. Could we close this?
